### PR TITLE
Update roslyn-sdk analyzer sample

### PR DESCRIPTION
**Pull request by Youssef1313**
Contributes to #4378

### CODE REVIEW

File: csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```
Comment (BillWagner) [inline csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs:1]: I'm curious.

This file, and several others, have strange name (the + and the ` character.)  Is that intentional? And why?

**Review by BillWagner — COMMENTED** [review]
The code looks great @Youssef1313 

I had one comment on the filenames. Once we resolve that, this is great.

File: csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs (hunk 1 above)
Comment (Youssef1313) [reply csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs:1]: @BillWagner This is what the default template has. The naming for this is made the same as the symbol metadata name.

**Review by BillWagner — APPROVED** [review]
This LGTM.

Thanks for all this work @Youssef1313 It's really good work.

**Pull request merged**